### ISSUE TITLE
ch listes

### ISSUE THREAD

**Issue by cuv** [mcodutti/Logique1-Syllabus#33]
ex 9: indentations des commentaires

**Comment by mcodutti** [comment]
Techniquement pas facile.
J'attend parce qu'il me semble que cet exercice est voué à disparaître.

**Comment by lbeeckmans** [comment]
Oui, vu qu'on a introduit des fonctions qui permettent de manier des chaines, cet exercice me semble avoir fait son temps. Je propose de le remplacer par la question 3 de l'examen de logique de 2ème session (août 2013) qui demande de créer une classe texte (ou le texte est vu comme une liste de chaine). Cela ferait l'affaire je pense, et offre un exemple de création d'une petite classe pas trop compliquée où l'attribut principal est une liste.
